### ISSUE TITLE
BQ3 crash when clicking search button

### ISSUE THREAD

**Issue by Prometheus0000** [GTNewHorizons/GT-New-Horizons-Modpack#7719]
Clicking the search button crashed it.
[crash-2021-03-28_15.29.19-client.txt](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/6217922/crash-2021-03-28_15.29.19-client.txt)

This is the version from skiftcha that doesn't crash on deleting quests, but can't create quests properly.
And now it also crashed with the version in 2.1.0.5. :(

I know I'm basically the only one using this to make quests, but it's very annoying when I have to switch versions because each of them crashes in different ways. Pls fix.

**Comment by Dream-Master** [comment]
nope I had such issues too. After the mid dev skpro leaves idk who can take care.

**Comment by Prometheus0000** [comment]
A clean instance of the 2.1.0.5 modpack works. But after I loaded the changes to the QB I was making in my other instance, it crashes when trying to search. So I guess that the multiple crashes I had because of the previous two crashing bugs caused the questfile to become malformed. Any clues on how to fix this quickly? Or do I just gotta compare files?

**Comment by Prometheus0000** [comment]
I have determined that the cause of the crash is somewhere in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/7717/commits/10b6967dc195fa1e66fb6c277e169e9e32fc5318 so it might take a while to get there. Going to other versions does not fix/cause it, and reloading the QB to the previous commit causes it to not crash.

@Dream-Master are you sure you experienced the same crash? From hitting the search button and not the other two crash bugs I posted? Or #7282 you posted about? My changes aren't merged yet so they shouldn't affect anyone else yet. Also, where did you get the version of BQ3 in the 2.1.0.5 pack, its CRC doesn't match builds 43-46, and the version is different?
